coap-gateway/service/config.go

```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/plgd-dev/kit/sync/task/queue"
 )
 
+type DeviceStatusValidityConfig struct {
+	Enable   bool          `yaml:"enable" json:"enable"`
+	Validity time.Duration `yaml:"validity" json:"validity"`
```

Comment:
`ExpiresIn`

Reply:
Is this value somehow related to ping?
e.g. if user configures expiresIn to 5secodns but ping between device and cloud is set to 30 seconds?

Reply:
no it is not related to ping

Reply:
And shouldn't be? at least, it should be documented how it behaves with combination with ping.